### PR TITLE
translate.php refactoring and directory structure adjustments

### PR DESCRIPTION
**Pull request by following5**

### CODE REVIEW

File: htdocs/translate.php [hunk 1]
```diff
@@ -88,136 +88,149 @@
...
 
-// @todo migrate to switch case
+switch ($action) {
+    case 'selectlang':
+        break;
 
-if ($action == 'selectlang') {
-} else {
-    if ($action == 'verify') {
+    case'verify':
```
Comment (teiling88) [inline htdocs/translate.php]: please add a blank line to case' -> case '

File: htdocs/translate.php [hunk 2]
```diff
@@ -88,136 +88,149 @@
...
+        edit();
+        break;
+
+    case 'copy_en':
+        copy_english_texts();
+        break;
+
+    case 'listfaults':
+        $trans = sql(
+            "SELECT
```
Comment (teiling88) [inline htdocs/translate.php]: please use single quotes instead of " if it is possible

**Comment by teiling88** [comment]
good job following same really small things and we can merge it :-)

File: htdocs/translate.php (hunk 2 above)
Comment (following5) [inline htdocs/translate.php]: SQL statements throughout the OC code are enclosed in double quotes, while string constants within the SQLs use single quotes.

File: htdocs/translate.php (hunk 1 above)
Comment (following5) [inline htdocs/translate.php]: Done, thanks.

File: htdocs/translate.php (hunk 2 above)
Comment (teiling88) [inline htdocs/translate.php]: if i remember correctly, this was the only sql statement without a constant so i left a note. but it is not so important.

Comment (following5) [inline htdocs/translate.php]: There was no note, and I think we should keep this consistent and enclose all SQL statements in double quotes.

**Pull request merged**